docs/intro/basic-tutorial.rst

```diff
@@ -113,13 +113,16 @@ extract a property from the ``to_item`` method:
 Configuring the project
 =======================
 
-To use ``scrapy-poet``, enable its downloader middleware in ``settings.py``:
+To use ``scrapy-poet``, enable its middlewares in ``settings.py``:
 
 .. code-block:: python
 
     DOWNLOADER_MIDDLEWARES = {
         'scrapy_poet.InjectionMiddleware': 543,
     }
+    SPIDER_MIDDLEWARES = {
+        "scrapy_poet.RetrySpiderMiddleware": 275,
```

Comment:
It's a bit unclean that InjectionMiddleware is not named InjectionDownloaderMiddleware, or that RetrySpiderMiddleware is not named RetryMiddleware. But that's probably fine. What do you think?

Reply:
The confusion with the built-in middleware is not that much of a concern to me, I think the different modules make it clear enough.

I do not like that there is no way besides documentation to tell if the injection middleware is a downloader middleware or a spider middleware. In Scrapy built-in middlewares you can at least tell from the module path, but we do not have that here.

But yes, the name I chose breaks consistency, and it is probably OK to name it `RetryMiddleware`. I will do that.

Reply:
Oh, I had forgotten that after these changes we _are_ separating them into different files. I am feeling much better with the name change :smile: